dubbo-common/src/main/java/org/apache/dubbo/config/AbstractInterfaceConfig.java

```diff
@@ -252,7 +252,7 @@ protected void postProcessAfterScopeModelChanged(ScopeModel oldScopeModel, Scope
         }
         if (CollectionUtils.isNotEmpty(this.registries)) {
             this.registries.forEach(registryConfig -> {
-                if (registryConfig.getScopeModel() != applicationModel) {
+                if (registryConfig != null && registryConfig.getScopeModel() != applicationModel) {
```

Comment:
**Issue**: Adding null check for registryConfig to prevent potential NullPointerException. <br> **Fix**: Add a null check before accessing registryConfig properties. <br> **Code Suggestion**: 
 ```
 -                if (registryConfig.getScopeModel() != applicationModel) {
 +                if (registryConfig != null && registryConfig.getScopeModel() != applicationModel) {
 ```